### DISCUSSION TITLE
Upgrading grapes.js

### DISCUSSION THREAD

**Discussion by rw-miroslavstojanovic** [GrapesJS/grapesjs#4587] [Q&A]
when using build command, is it possible to generate grapes.js instead of grapes.min.js. We are using old version of grapes.js which include our custom code, so we would like to upgrade it.

**Comment by ronaldohoch** [comment]
I think you can diff the two repos 🤔
https://stackoverflow.com/questions/1968512/getting-the-difference-between-two-repositories

**Reply by rw-miroslavstojanovic** [reply]
We have the older one, the grapes repo that is 3 years old and from that repo we have grapes.js and we still use it. But the latest one (ver.v0.20.1) from the github repo, when we run build command and build the project it builds only grapes.min.js file, so I wonder if there is a way to build the grapes.js file? so we could do diffs with the older one. Thanks
